server.py

```diff
@@ -0,0 +1,41 @@
+from flask import Flask
+import json
+
+app = Flask(__name__)
+
+
+@app.route('/status')
+def status():
+    return json.dumps({'status': 'online'})
+
+
+@app.route('/')
+def index(): pass
+
+
+@app.route('/login')
+def login(): pass
+
+
+@app.route('/user/<username>')
```

Comment:
The username might not be safe to put in a URL, so we should use the numeric ID here

Reply:
Why is this unsafe if the username will be visible on the website? The API should be able to filter by public username